### PR TITLE
[#92] index AVUs applied via iput --metadata (4-2-stable)

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

File: libirods_rule_engine_plugin-indexing.cpp [hunk 1]
```diff
@@ -201,6 +201,14 @@ namespace {
                     object_path,
                     _rei->rsComm->clientUser.userName,
                     source_resource);
+
+                const char* metadata_included = getValByKey(&obj_inp->condInput, METADATA_INCLUDED_KW);
+                if (metadata_included)
+                {
+                    idx.schedule_metadata_indexing_event(
+                        object_path,
+                        _rei->rsComm->clientUser.userName, "a", "v", "u"); // the actual AVU values do not matter since commit 277933912f
```
Comment (korydraughn) [inline libirods_rule_engine_plugin-indexing.cpp:210]: Does the commit referenced in the comment point to a commit in 4-2-stable?

File: packaging/test_plugin_indexing.py [hunk 2]
```diff
@@ -36,6 +36,10 @@
     long_type = long # Python 2.x
 except NameError: pass
 
+Manual_Tests_Enabled = (
+    os.environ.get("MANUALLY_TEST_INDEXING_PLUGIN","")[:1].lower() not in ("n", "f", "0", "")
+    )
```
Comment (korydraughn) [inline packaging/test_plugin_indexing.py:41]: Is this documented anywhere?

**Review by korydraughn — COMMENTED** [review]
Code looks fine.

File: libirods_rule_engine_plugin-indexing.cpp (hunk 1 above)
Comment (d-w-moore) [reply libirods_rule_engine_plugin-indexing.cpp]: referring to new issue instead

File: packaging/test_plugin_indexing.py (hunk 2 above)
Comment (d-w-moore) [reply packaging/test_plugin_indexing.py]: Yes, just did that. README section on Testing the Plugin

**Comment by d-w-moore** [comment]
Squashed; tested on C7 as well.
Not sure why all the objections to the new markup in README.  Codacy seemed fine with the very same changes made in `main` branch...

File: libirods_rule_engine_plugin-indexing.cpp (hunk 1 above)
Comment (korydraughn) [reply libirods_rule_engine_plugin-indexing.cpp]: Please mark as resolved.

**Comment by d-w-moore** [comment]
#'d

**Review by trel — APPROVED** [review]

**Review by alanking — APPROVED** [review]

**Pull request merged**